druid/examples/either.rs

```diff
@@ -12,42 +12,43 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! An example using the Either widget to show/hide a slider.
+//! This is a very simple example, it uses a bool to determine
+//! which widget gets shown.
+
+use druid::widget::prelude::*;
 use druid::widget::{Checkbox, Either, Flex, Label, Slider};
-use druid::{AppLauncher, Data, Lens, LocalizedString, Widget, WidgetExt, WindowDesc};
+use druid::{AppLauncher, Data, Lens, WidgetExt, WindowDesc};
 
 #[derive(Clone, Default, Data, Lens)]
 struct AppState {
     which: bool,
     value: f64,
 }
 
-pub fn main() {
-    let main_window = WindowDesc::new(ui_builder).title(
-        LocalizedString::new("either-demo-window-title")
-            .with_placeholder("Switcheroo")
-            .with_arg("view", |data: &AppState, _env| (data.which as u8).into()),
-    );
-    let data = AppState::default();
-    AppLauncher::with_window(main_window)
-        .use_simple_logger()
-        .launch(data)
-        .expect("launch failed");
-}
-
 fn ui_builder() -> impl Widget<AppState> {
-    let label = Label::new("Click to reveal slider");
+    // Our UI consists of a column with a button and an `Either` widget
+    let button = Checkbox::new("Toggle slider")
+        .lens(AppState::which)
+        .padding(5.0);
 
-    let mut col = Flex::column();
-    col.add_child(
-        Checkbox::new("Toggle slider")
-            .lens(AppState::which)
-            .padding(5.0),
-    );
+    // The either widget has 2 children, one of which is visible.
+    // you have to pass in a closure which takes `Data` and and `Env`.
+    // This closure determines which widget gets displayed based on the
+    // return value (a bool). False is the second widget and true is the first.
```

Comment:
```suggestion
    // The `Either` widget has two children, only one of which is visible at a time.
    // To determine which child is visible, you pass it a closure that takes the
    // `Data` and the `Env` and returns a bool; if it returns `true`, the first
    // widget will be visible, and if `false`, the second.
```

english convention varies, but a good guideline for technical writing is to write numbers under 10 (so up to nine, inclusive 😉) as words.

Reply:
If my writing is very bad let me know, I can put more effort into it.
I don't have a preference here but maybe you view that completely different.